### DISCUSSION TITLE
В чем отличие версий библиотек ESP32 от Espressif System v 1. и v 2.

### DISCUSSION THREAD

**Discussion by psasrg71** [espressif/arduino-esp32#9035] [Q&A]
При обновлении библиотеки с версии v.1.0.6 на v.2.  при компиляции возникает ошибка в строке WiFi.onEvent(WiFiPointConnected, SYSTEM_EVENT_AP_STACONNECTED).  Подскажите что делать? Хотелось бы использовать старый код с обновленной библиотекой

**Comment by me-no-dev** [comment]
We all communicate in English. Not sure what you expect when posting in Russian?
